src/py_eddy_tracker/dataset/grid.py

```diff
@@ -2541,6 +2551,11 @@ def get_uv_quad(i0, j0, u, v, m, nb_x=0):
     i1, j1 = i0 + 1, j0 + 1
     if nb_x != 0:
         i1 %= nb_x
+    i_max, j_max = m.shape
```

Comment:
for which use?

Reply:
mistake here, i did comment line 2556 & 2557 but they should not be.
Bug here when u,v data are not wide enough to cover all space taken by particles, and particule is advected on edge of u,v matrix. i0 can me > i_max